### PR TITLE
MLB showdown

### PR DESCRIPTION
**Pull request by BenBrostoff**
~~Note - work in progress~~

Left to do:
- [x] Deploy new version
- [x] Add documentation
- [x] Add repl.it showdown example

### CODE REVIEW

File: draftfast/orm.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def __contains__(self, player):
                 if p.name == player or p.short_name == player:
                     return True
         elif isinstance(player, Player):
-            return p in self.players
+            return self.players
```
Comment (BenBrostoff) [inline draftfast/orm.py]: @sharkiteuthis Not sure if you wrote this piece but assuming this change was the intent?

Comment (sharkiteuthis) [reply draftfast/orm.py]: __contains__() is supposed to return a boolean, I think this change will break NFL if the string comparison short circuit doesn't return true

Comment (BenBrostoff) [reply draftfast/orm.py]: I think the issue here is `p` was out of scope originally. Maybe this?:

```python
    def __contains__(self, player):
        if isinstance(player, str):
            for p in self.players:
                if p.name == player or p.short_name == player:
                    return True
        elif isinstance(player, Player):
            return player in self.players
        else:
            raise NotImplementedError
```

File: draftfast/rules.py [hunk 2]
```diff
@@ -131,6 +133,10 @@ def get_nfl_showdown_positions(dk: bool = False, fd: bool = False) -> list:
         'NHL_SHOWDOWN': [
             ['FLEX', 6, 6],
         ],
+        'MLB_SHOWDOWN': [
+            ['CPT', 1, 1],
```
Comment (BenBrostoff) [inline draftfast/rules.py]: This is an issue I think for all the other showdown games. It at least impacts the display if `CPT` isn't a constraint here - everything shows as `FLEX`.

Comment (sharkiteuthis) [reply draftfast/rules.py]: yeah, i think NHL showdowns are broken or at least non-optimal right now. Look at `get_nfl_showdown_positions`  or `NBA_SHOWDOWN` rules

File: draftfast/showdown/orm.py [hunk 3]
```diff
@@ -13,8 +13,6 @@ def __init__(self, player: Player, captain: bool = False):
             self.real_pos = self.pos
             self.pos = 'CPT'
             self.captain = True
-            self.proj *= 1.5
```
Comment (BenBrostoff) [inline draftfast/showdown/orm.py]: I think this is a change we want. I question hard-coding a 1.5 - the drawback I guess is that the rules of DraftKings are no longer "exposed" anywhere - but if we are going to expose them, it should be done as something you can pass into the initializer or maybe in `RuleSet`.

Comment (sharkiteuthis) [reply draftfast/showdown/orm.py]: I thought per #140 the CSVs provided by DK include CPT entires?

Comment (BenBrostoff) [reply draftfast/showdown/orm.py]: They do - I think I want to keep `ShowdownPlayer` though since `real_pos` is valuable and used by the optimizer.

**Pull request merged**